first_order_model/shrink_util.py

```diff
@@ -872,7 +935,6 @@ def channel_prune(model, deletions):
                 
```

Comment:
Yes, this gets taken care of because the depthwise portion is considered "tied" in that if the previous layer's outputs change (i.e. this ones input changes), then the depthwise layer is "tied" to the change and must also change its output.